dir.traversal.targets

```diff
@@ -6,7 +6,7 @@
     <!-- To Serialize we use msbuild's batching functionality '%' to force it to batch all similar projects with the same identity 
          however since the project names are unique it will essentially force each to run in its own batch -->
     <MSBuild Targets="Build" Projects="@(Project)" Condition="'$(SerializeProjects)'=='true'" Properties="Dummy=%(Identity)"/>
-    <MSBuild Targets="Build" Projects="@(Project)" Condition="'$(SerializeProjects)'!='true'" BuildInParallel="true" />
+    <MSBuild Targets="Build" Projects="@(Project)" Condition="'$(SerializeProjects)'!='true'" BuildInParallel="false" />
```

Comment:
Where are we getting msbuild when building on Unix?  Is it coming down with BuildTools?  If so, can we control the version independently?  If we're being forced to take the VS 2017 version, we may need this workaround for now.

@rartemev Is the problem easy to repro by simply reverting the change to this line?  Can you share the error you're seeing?  We need to get this info to the MSBuild team.